.bundle/config

```diff
@@ -0,0 +1,5 @@
+---
+BUNDLE_RETRY: "3"
+BUNDLE_PATH: "vendor/bundle"
+BUNDLE_JOBS: "3"
+BUNDLE_WITHOUT: "screenshots"
```

Comment:
This made me realize a side effect of this: when we want to generate screenshots, we will run `bundle install --with screenshots` – or, as it's nowadays suggested by bundler instead, `bundle config --local --delete without` (see [this in WPAndroid for example](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/.circleci/config.yml#L460-L462)

In both cases this will have the effect of modifying this `bundle/config` file to renove this `BUNDLE_WITHOUT`, a change which should NOT be commited (as we don't want everyone to install gems that are useless for most people if they never have to create screenshots in most cases).

This is quite unlikely to happen as most devs will never have to do that if screenshots are generated by an MIE or on CI, but something to keep in mind still…

Reply:
Yeah, this is true. It's kinda suboptimal no matter which way you go – we don't want people having to remember to run `bundle install --without screenshots` (or whatever the new syntax is 😅) 

But we also don't want to dirty the file every time. 

IMHO this is the _best_ option because as you note it'll only affect MIEs and not everybody else 😄

Once we drop ruby for all the things and only MIEs are using it, IMHO we could just remove the `without` part and make it a requirement if you want to use Ruby?